lensless/admm.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self._tau = tau
 
         # call reset() to initialize matrices
-        super(ADMM, self).__init__(psf, dtype, pad=False, norm="backward")
+        super(ADMM, self).__init__(psf, dtype, pad=False, norm="backward", **kwargs)
```

Comment:
bug fix so `n_iter` also gets passed?

Reply:
Yes. And generally, it is the proper way to call super.__init__.